File: exercism/python/raindrops/raindrops.py
sound_factors = {
    3: 'Pling',
    5: 'Plang',
    7: 'Plong',
}

def convert(number):
    sounds = [sound for factor, sound in sound_factors.items() if number % factor == 0]

    return ''.join(sounds) or str(number)

File: exercism/go/raindrops/raindrops.go
package raindrops

import "strconv"

// Convert a numbers factors into raindrop sounds
func Convert(num int) string {
	factorSounds := map[int]string{
		3: "Pling",
		5: "Plang",
		7: "Plong",
	}

	raindropSounds := ""
	for factor, sound := range factorSounds {
		if num%factor == 0 {
			raindropSounds += sound
		}
	}

	if raindropSounds == "" {
		return strconv.Itoa(num)
	}

	return raindropSounds
}
